accounts/abi/bind/base.go

```diff
@@ -254,7 +256,7 @@ func (c *BoundContract) createDynamicTx(opts *TransactOpts, contract *common.Add
 	if gasFeeCap == nil {
 		gasFeeCap = new(big.Int).Add(
 			gasTipCap,
-			new(big.Int).Mul(head.BaseFee, big.NewInt(2)),
+			new(big.Int).Mul(head.BaseFee, big.NewInt(elasticityMultiplier)),
```

Comment:
I mean, here, without defining a new const.

Reply:
Attempted to update to use `params. elasticityMultiplier` but it seems `params` is not avaialble in this local context. Would it be your advice to add `params` all the way through the call-stack?